core/src/io/anuke/mindustry/ui/dialogs/SettingsMenuDialog.java

```diff
@@ -228,6 +228,7 @@ void addSettings(){
         game.checkPref("blockreplace", true);
         game.checkPref("conveyorpathfinding", true);
         game.checkPref("hints", true);
+        game.checkPref("buildautopause", false);
```

Comment:
This should only be added/displayed `if(!mobile)`, since mobile doesn't have build pausing.